### DISCUSSION TITLE
Text Selection Action Bar On Android & IOS

### DISCUSSION THREAD

**Discussion by llama-nl** [godotengine/godot-proposals#10979] [GUI]
### Describe the problem or limitation you are having in your project

actually thing:
![Screenshot_20241015_100432](https://github.com/user-attachments/assets/b05fea75-8498-4026-91fe-d9478574f02f)

expection:
![Screenshot_20241015_100456_One UI Home](https://github.com/user-attachments/assets/df3cdc14-2743-4141-bdaa-d96c42c05a83)

In Godot's text editors on Android/iOS, text selection doesn't function like it does in typical mobile applications. Instead, the default behavior mirrors that of desktop platforms. Ideally, when selecting text, I would expect to see the standard text selection action bar, as is common in mobile environments.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

It should work like regular Android or IOS application. Like if we double tap on a text the test will be get selected and a action bar will appear.

**Comment by llama-nl** [comment]
And also I strongly urge the community to prioritize these platforms. Thanks.

**Comment by Calinou** [comment]
Moving to discussion, as there is no concrete technical implementation provided.

PS: The questions in the proposal form are not optional; **all of them** must be answered (and not just with "N/A"). If you have difficulty answering any of them, please open a discussion instead (this proposal has already been converted to a discussion).

**Comment by Calinou** [comment]
Proposal issue opened: https://github.com/godotengine/godot-proposals/issues/11028
